app/src/main/java/org/p2p/wallet/swap/ui/jupiter/tokens/presenter/SwapTokensPresenter.kt

```diff
@@ -57,7 +58,9 @@ class SwapTokensPresenter(
                 )
             }
         }
+        // todo: optimize. There are about 800~ items being set at once, we need to set list partially
```

Comment:
is this a perfomance problem? i thought recyclerview handles this via viewholders